glide.yaml

```diff
@@ -46,6 +46,10 @@ import:
 - package: github.com/gorilla/websocket
   version: 0647012449a1878977514a346b26637dd022446c
 - package: google.golang.org/grpc
+- package: k8s.io/apiextensions-apiserver
+  version: 4b903ca6c8a2031b8f88c3d1fbefa8eb248b103c
+- package: github.com/ant31/crd-validation
```

Comment:
Not entirely clear to me what exactly this one does. Want to look into it a little bit more.

Reply:
crd-validation is a small package that a coreos guy made. That package uses the openapi generator code to generate just the schema for a single object.

I had actually started writing the exact same code this package implements before I found it. If the package actually involved our runtime, I wouldn't have considered it, but since it's just build time logic I'm willing to include it. 

Just look at the function I actually use from this package in the tools/crd-generator/crd-generator.go tool.